### PR TITLE
Add `getSfu()`

### PR DESCRIPTION
**Pull request by bogue**
Add the ability to retrieve SFU state without throwing an error

### CODE REVIEW

File: src/simple/Phone.js [hunk 1]
```diff
@@ -221,9 +221,14 @@ class Phone extends Emitter {
     });
   }
 
-  checkSfu() {
-    const hasSfu = this.sipLine.options.some(option =>
+  getSfu() {
+    return this.sipLine && this.sipLine.options.some(option =>
       option[0] === 'max_audio_streams' || option[0] === 'max_video_streams');
```
Comment (sboily) [inline src/simple/Phone.js:226]: Not sure about this check, we need to have more than one max_video_streams and more than zero for max_audio_streams.

Comment (bogue) [reply src/simple/Phone.js:226]: Noted. 

Note: closing this PR, we'll be handling the SFU check directly in WDA for now

**Comment by sboily** [comment]
Please fix test.

File: src/domain/SipLine.js [hunk 2]
```diff
@@ -41,6 +43,8 @@ export default class SipLine {
   links: Array<Object>;
   trunk: ?string;
   line: Endpoint;
+  hasVideo: boolean;
+  hasVideoConference: boolean;
```
Comment (manuquentin) [inline src/domain/SipLine.js:47]: Can we have a getter for these 2 attributes ? 
Can you update the documentation accordingly ?

**Pull request merged**